coursebuilder/app.yaml

```diff
@@ -1,4 +1,4 @@
-application: mycourse  # FIXME: Replace this with your application id
+application: cfafrica-mooc-dev  # FIXME: Replace this with your application id
```

Comment:
@Sharonsyra you may want not to commit this as it's going to change from deployment to deployment.

Reply:
Thanks i will fix that.